lib/stages/FEngine.cpp

```diff
@@ -414,19 +417,7 @@ FEngine::FEngine(kotekan::Config& config, const std::string& unique_name,
         julia_source.at(julia_source_length) = '\0';
         kotekan::juliaCall([&]() {
             jl_value_t* const res = jl_eval_string(julia_source.data());
-            jl_value_t* exc = jl_exception_occurred();
-            if (exc) {
-                FATAL_ERROR("Caught Julia exception");
-
-                // Get Base.showerror
-                jl_function_t* showerror = jl_get_function(jl_base_module, "showerror");
-
-                // Call showerror(io, exc)
-                jl_call2(showerror, jl_stderr_obj(), exc);
-
-                // jl_flush_cstdio()
-                // jl_print_backtrace()
-            }
+            kotekan::juliaHandleExceptions();
```

Comment:
Hmm, in this case the should be called "juliaHandlePossibleExceptions"? Since it also covers the case where no exception happened?

Reply:
It also handles the "there is no exception" case though, which at least to me, was surprising.